### DISCUSSION TITLE
Can I use this with .NET Core?

### DISCUSSION THREAD

**Discussion by thepwrtank18** [ookii-dialogs/ookii-dialogs-winforms#23] [Q&A] [answered]
Using .NET Core 3.5, and it gives me a warning that it only supports specific .NET Framework versions.

If I can't, are there any alternatives?

**Comment by augustoproiete** [comment, marked as answer]
@thepwrtank18 Do you mean .NET Core 3.1? Or do you mean .NET Framework 3.5?

If you mean .NET Core 3.1, the code in the repo should work well on .NET Core 3.1 and .NET 5. I haven't yet had a chance to release it to NuGet yet, but planning to do so over the next few days.

If you mean .NET Framework 3.5, then no... Ookii Dialogs require at minimum .NET Framework 4.5

**Reply by TRCarbone** [reply]
I am converting a .NET Framework 4.8 project to .NET 5. Is it possible you can release a NuGet package for this?

**Reply by augustoproiete** [reply]
Sure. Will get it published on nuget.org in the next few days

**Reply by TRCarbone** [reply]
Still looking for this. Hopefully soon? :)

**Reply by augustoproiete** [reply]
@TRCarbone  Ookii.Dialogs.WinForms [v3.0.0](https://github.com/ookii-dialogs/ookii-dialogs-winforms/discussions/38) was just released and adds support for .NET 5 as well as .NET Core 3.1 - https://github.com/ookii-dialogs/ookii-dialogs-winforms/discussions/38

**Comment by thepwrtank18** [comment]
.NET 3.1. Thanks for the info.

**Reply by augustoproiete** [reply]
@thepwrtank18 Ookii.Dialogs.WinForms [v3.0.0](https://github.com/ookii-dialogs/ookii-dialogs-winforms/discussions/38) was just released and adds support for .NET 5 as well as .NET Core 3.1 - https://github.com/ookii-dialogs/ookii-dialogs-winforms/discussions/38
